src/client/lazy-app/Modal/style.css

```diff
@@ -0,0 +1,200 @@
+dialog {
+  padding: 0;
+  width: 70vw;
+  max-width: 60ch;
+  max-height: 70vh;
+
+  box-shadow: 0 0 20px 0 rgba(0, 0, 0, 20%);
+
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+
+  font-size: 1.5em;
+  color: var(--light-gray);
+
+  border: none;
+  border-radius: 10px;
+  overflow: hidden;
+
+  background-color: var(--off-black);
+
+  opacity: 0;
+  transform: translateY(50px);
+  transition: 250ms ease;
+
+  /* backdrop can't be transitioned easily, so we must   */
+
+  &::backdrop {
+    background-color: rgba(0, 0, 0, 30%);
+  }
+
+  &[open]::backdrop {
+    animation: backdrop-fade-in 250ms linear;
+
+    @keyframes backdrop-fade-in {
+      from {
+        opacity: 0;
+      }
+      to {
+        opacity: 1;
+      }
+    }
+  }
+
+  &.modal--closing::backdrop {
+    transition: opacity 250ms linear;
+    opacity: 0;
+  }
+}
+
+dialog[open] {
+  opacity: 1;
+  transform: translateY(0px);
```

Comment:
@argyleink I'm a bit confused, you suggested 20px to show intent - but wouldn't it be better to add a comment here instead of changing the translation amount? 20px isn't making much of a visual difference on mobile IMO

Reply:
sorry for the confusion!

this
![Screenshot 2023-07-10 at 12 20 45 PM](https://github.com/GoogleChromeLabs/squoosh/assets/1134620/059873d0-2ea6-4739-a7e6-b8b987ce15fb)

is better than this
![Screenshot 2023-07-10 at 12 20 33 PM](https://github.com/GoogleChromeLabs/squoosh/assets/1134620/8b730d3d-9894-4078-bc27-805bab41a262)

is what I'm getting at. the intent is to have a modal, and it's weird if it's stuck to the top flush, better if it's got some breathing room from the top edge.

Reply:
Huh, that's weird - it's supposed to be centered on all devices/sizes, not stuck to the top. Could you tell me which browser/os you're using? I've tested it on most Android browsers and remotely tested on a couple iOS versions too, seemed to work fine on those.

Reply:
Chrome stable 114

Reply:
confirm devtools issue